src/client/src/components/Input/index.tsx

```diff
@@ -5,6 +5,7 @@ import styles from "./styles.module.css";
 
 interface IProps {
   handleChange: (e: React.SyntheticEvent<HTMLInputElement>) => any;
+  handleKeyDown?: (e: React.KeyboardEvent<HTMLInputElement>) => any;
```

Comment:
might want to set to void since you're not returning anything from the function

Reply:
done